src/common/TestsRunner.ts

```diff
@@ -4,12 +4,14 @@
 
 import { Counter } from 'prom-client';
 
-import { ITest, TestResults } from '../tests/Test';
+import { ITest } from '../tests/Test';
 import { IMetrics, StopTimerFunction } from './Metrics';
 
 const HISTOGRAM_NAME: string = 'monitoring_test';
 const COUNTER_NAME: string = 'monitoring_test_fails';
 
+type Status = 0 | 1;
```

Comment:
```suggestion
type Status = 'success' | 'failure';
```

WDYT about using string names for statuses because:
- label values are strings
- we assumed that alerts/charts can be modified manually in Grafana, so it will be easier to understand what specific status means. For people who don't know shell scripting status `1` could mean "success" 😅 
- inspiring with [Gradle test results enum](https://docs.gradle.org/current/javadoc/org/gradle/api/tasks/testing/TestResult.ResultType.html) I realized that in the future there can be a case when we would like to add another status like `skipped` 🤔